src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/VisualBasicSymbolMatcher.vb

```diff
@@ -692,6 +692,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             End Function
 
             Public Overrides Function VisitNamedType(type As NamedTypeSymbol) As Symbol
+                If (type.IsTupleType) Then
```

Comment:
thanks!

---
In reply to: [116131000](https://github.com/dotnet/roslyn/pull/19283#discussion_r116131000) [](ancestors = 116131000)